platform/android/export/gradle_export_util.h

```diff
@@ -55,6 +55,7 @@ static const int APP_CATEGORY_NEWS = 5;
 static const int APP_CATEGORY_PRODUCTIVITY = 6;
 static const int APP_CATEGORY_SOCIAL = 7;
 static const int APP_CATEGORY_VIDEO = 8;
+static const int APP_CATEGORY_OTHER = 9;
```

Comment:
Let's use `CATEGORY_UNDEFINED` to match the documentation

https://developer.android.com/reference/android/content/pm/ApplicationInfo?hl=en#CATEGORY_UNDEFINED